src/main/java/by/sanko/servlets/SearchServlet.java

```diff
@@ -0,0 +1,69 @@
+package by.sanko.servlets;
+
+import by.sanko.repo.Posts;
+import by.sanko.repo.Tweet;
+
+import javax.servlet.ServletException;
+import javax.servlet.ServletOutputStream;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+
+@WebServlet(name = "SearchServlet")
+public class SearchServlet extends HttpServlet {
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        String id = request.getParameter("id");
+        String createdAt = request.getParameter("createdAt");
+        String rank = request.getParameter("rank");
+        String nation = request.getParameter("nation");
+        String author = request.getParameter("author");
+        ServletOutputStream out = response.getOutputStream();
+        int i = 0;
+        String[] param = new String[5];
```

Comment:
You should have used `List<String>` here. This is designed to hande variable-length collections